test/app.js

```diff
@@ -797,8 +797,9 @@ t.test('App', async t => {
 
     t.notOk(app.validator.schema('test123'));
     (await ua.putOk('/schema/dynamic', {json: {test: 123}})).statusIs(200).jsonIs({valid: true, errors: []});
+    (await ua.putOk('/schema/dynamic', {json: {test: ' 123'}})).statusIs(200).jsonIs({valid: true, errors: []});
     t.ok(app.validator.schema('test123'));
-    (await ua.putOk('/schema/dynamic', {json: {test: '123'}})).statusIs(200).jsonIs({
```

Comment:
Why remove a perfectly good test case?

Reply:
This test would not pass, because <test: '123'> is coerced to <test: 123>, so will be considered valid

Reply:
I don't see why that would mean it shouldn't be tested anymore.

Reply:
it is tested, just before this one, but the expected result is valid: true.

for the valid: false case, the input is {test: 'a123'}, so it cannot be coerced to a number

Reply:
The one before this tests whitespace trim, but there is no pure string to number test.